apps/desktop-wallet/src/features/send/sentTransactions/SentTransactionSnackbarPopup.tsx

```diff
@@ -112,12 +119,21 @@ const Progress = ({ status }: Pick<SentTransaction, 'status'>) => {
     }
   }, [status])
 
-  useInterval(() => setProgress((prevValue) => prevValue + 0.015), 1000, status === 'confirmed' || progress > 0.9)
+  useInterval(
+    () => setProgress((prevValue) => Math.min(prevValue + 0.015, 1)),
```

Comment:
I don't understand the purpose of this `Math.min`, since we stop it at 0.9?

Reply:
You're right! Tried another approach, then forgot to remove the `min` function!

Reply:
Fixed